app/presenters/weather_presenter.rb

```diff
@@ -0,0 +1,40 @@
+class WeatherPresenter
+  def initialize(data)
+    @data = data
+  end
+
+  def description
+    data['current']['condition']['text']
+  end
+
+  def temperature
+    data['current']['temp_c']
+  end
+
+  def icon
+    data['current']['condition']['icon']
```

Comment:
tutaj wskazówka: zawsze jak coś nam się powtarza w kodzie, to warto sobie zadać pytanie, czy nie można wrzucić tej powtarzającej się części kodu do metody prywatnej. Tutaj prosi się o metodę `current_data`